scripts/variantstore/tieout/add_max_as_vqslod.py

```diff
@@ -19,33 +19,30 @@
 
         parts = line.split("\t")
         
-        if ("ExcessHet" in parts[6]):
+        # strip out hard filtered sites, so vcfeval can use "all-records" to plot the ROC curves
```

Comment:
Be sure to remember to reindex any files you run this on...

Reply:
Yep -- this is only used from a benchmarking pipeline (see scripts/variantstore/tieout/GIAB_TIEOUT.md) and it runs tabix to generate the index